website/build_website.sh

```diff
@@ -0,0 +1,17 @@
+#
+# This script generates documentation using pydoc-markdown and renders the website using Quarto.
+#
+# Usage: bash build_website.sh
+#
+
+# Generate documentation using pydoc-markdown
+pydoc-markdown
+
+# Render the website using Quarto
+quarto render ./docs
+
+# Process notebooks using a Python script
+python ./process_notebooks.py
+
+# Start the website using yarn
+yarn start
```

Comment:
While this is definitely helpful, it adds another place to keep in sync

Reply:
It does. How about we keep this up to date and speed us up. And fix the redundant text in website/readme and contribute.md. That way there are only two things:
- script 
- documentation of the script

Thoughts?

Reply:
Yeah that sounds good!